pylint/utils/pragma_parser.py

```diff
@@ -9,17 +9,19 @@
 # so that an option can be continued with the reasons
 # why it is active or disabled.
 OPTION_RGX = r"""
-    \s*                # Any number of whitespace
-    \#?                # One or zero hash
-    .*                 # Anything (as much as possible)
-    (\s*               # Beginning of first matched group and any number of whitespaces
-    \#                 # Beginning of comment
-    .*?                # Anything (as little as possible)
-    \bpylint:          # pylint word and column
-    \s*                # Any number of whitespaces
-    ([^;#\n]+))        # Anything except semicolon or hash or newline (it is the second matched group)
-                       # and end of the first matched group
-    [;#]{0,1}"""  # From 0 to 1 repetition of semicolon or hash
+    (?:^\s*\#.*           # Comment line or
+      |\s*                # any number of whitespaces or
+      |\s*\#.*            # a beginning of line comment
```

Comment:
```suggestion
    (?:^\s*\#.*           # Line with only a comment,
      |\s*                # or any number of whitespaces,
      |\s*\#.*            # or the beginning of a comment
```

Please correct if this is wrong, but this is a little clearer imo.

Reply:
Well, this more or less for consistency with the rest (every line has a comment).  I'm +1 for removing obvious ones (e.g. L12-13 too).

Reply:
For me only the difference between L12 and L14-15 warrants a comment, as from a quick glance at the pattern you don't immediately see (or at least I don't 😅) what the difference between them is.
Agree that L13 is redundant. (If `|\s*` is unknown to you perhaps you shouldn't bother with this pattern yet 😄)